lib/ffprobe.js

```diff
@@ -163,26 +163,28 @@ module.exports = function(proto) {
 
           // Handle legacy output with "TAG:x" and "DISPOSITION:x" keys
           [data.format].concat(data.streams).forEach(function(target) {
-            var legacyTagKeys = Object.keys(target).filter(legacyTag);
+            if (target) {
```

Comment:
This is another fix, sometimes target is null, which would throw an exception and crash node.